### PR TITLE
Docs issue #1038 roles for node port with xsite url

### PR DESCRIPTION
**Pull request by oraNod**
#1038

### CODE REVIEW

**Review by dmvolod — APPROVED** [review]

**Comment by ryanemerson** [comment]
Thanks @oraNod

File: documentation/asciidoc/topics/proc_creating_sa_tokens.adoc [hunk 1]
```diff
@@ -25,6 +25,13 @@ serviceaccount/lon created
 $ oc policy add-role-to-user view system:serviceaccount:<namespace>:lon
 ----
 +
+. If you use a node port service to expose {brandname} clusters on the network, you must also add the `cluster-reader` role to the service account:
```
Comment (rigazilla) [inline documentation/asciidoc/topics/proc_creating_sa_tokens.adoc:28]: Isn't this needed also for k8s?

Comment (rigazilla) [reply documentation/asciidoc/topics/proc_creating_sa_tokens.adoc:28]: oh it's already merged :)

Comment (oraNod) [reply documentation/asciidoc/topics/proc_creating_sa_tokens.adoc:28]: @dmvolod can you advise please? I should've asked this myself tbf but I thought that the cluster-reader role was specific to OCP. I looked through k8s RBAC docs but didn't see it.

Comment (dmvolod) [reply documentation/asciidoc/topics/proc_creating_sa_tokens.adoc:28]: @rigazilla we have this rights in the ClusterRoleBinding and ClusterRole already

Comment (dmvolod) [reply documentation/asciidoc/topics/proc_creating_sa_tokens.adoc:28]: @oraNod please see documentation about this role https://docs.openshift.com/container-platform/4.6/authentication/using-rbac.html
```
16:11 $ oc get clusterroles | grep cluster-reader
cluster-autoscaler-operator:cluster-reader                  2021-05-16T04:47:46Z
cluster-reader                                                              2021-05-16T04:42:57Z
```

**Pull request merged**